services/controller/src/plz/controller/configuration.py

```diff
@@ -98,20 +98,21 @@ def _images_from(config, docker_host):
         client_extra_args = {'timeout': docker_api_client_timeout * 60}
     else:
         client_extra_args = {}
-    docker_api_client = docker.APIClient(
-        base_url=docker_host, **client_extra_args)
+
+    def docker_api_client_creator():
+        return docker.APIClient(base_url=docker_host, **client_extra_args)
```

Comment:
Might be nicer to just use lambdas for these.

```python
    docker_api_client_creator = lambda: docker.APIClient(base_url=docker_host, **client_extra_args)
```

Reply:
It seems it's bad practice according to the Intellij linter.

Reply:
M’kay, cool.